### ISSUE TITLE
return values not working correctly

### ISSUE THREAD

**Issue by geramy92** [EmmyLua/IntelliJ-EmmyLua#336]
Environment
--------------
   name   |   version  
----------|----------
 IDEA version         | 2019.3.2]
 EmmyLua version | 1.3.2.142-IDEA182
 OS                        | Windows 10]

What are the steps to reproduce this issue?（重现步骤？）
------------------------------------------------
1. see screenshot

What happens?（出现什么问题？）
-------------
Quick documentation only shows first return and also not both value datatype possibilities (in screenshot it shows only first issue)

What were you expecting to happen?（期望？）
-----------------------------------------
shows both returns and all types

Any logs, error output, etc?（有没有什么log, error输出？）
------------------------------
![lua issues](https://user-images.githubusercontent.com/2132554/74157360-72f7f400-4c18-11ea-815b-5a4d23b38e9d.png)

**Comment by winterwolf** [comment]
I agree, this is a bug. You can use such a crutch until it is fixed:

![screenshot-001](https://user-images.githubusercontent.com/42620439/74683030-5f2c2f00-51d0-11ea-9a06-948bfe444aa5.png)

**Comment by winterwolf** [comment]
btw, I think, `any|nil` in your example is incorrect, because nil **is** any. Everything is any :)

**Comment by winterwolf** [comment]
![screenshot-001](https://user-images.githubusercontent.com/42620439/74686731-129a2100-51db-11ea-99c2-325ff2f3398f.png)


I think, any|nil in your example is incorrect, because nil is any. Everything is any :)

**Comment by geramy92** [comment]
thanks this works, it's just in that kind of structure a bit bad, that you can not add extended description of return values - but anyways better as nothing :D

**Comment by dnchu** [comment]
Descriptions for return values are possible, no?

`@return boolean, any @ Status, Value`
